algorithms/scripts/SGDClassifier-master.py

```diff
@@ -35,7 +35,7 @@
 # In[3]:
 
 # We're going to be building a 'TP53' classifier 
-GENE = 'TP53'
+GENE = '7157' # TP53
```

Comment:
Is `algorithms/scripts/SGDClassifier-master.py` the nbconverted `2.TCGA-MLexample.ipynb`? It is difficult to compare the diffs between `1.TCGA-MLexample.ipynb` and `2.TCGA-MLexample.ipynb`